src/internal_events/file.rs

```diff
@@ -319,6 +320,7 @@ mod source {
                 counter!(
                     "files_unwatched_total", 1,
                     "file" => self.file.to_string_lossy().into_owned(),
+                    "reached_eof" => if self.reached_eof { "true" } else { "false" }
```

Comment:
Can you please add `"reached_eof"` to the info log above and also to the counter emitted in the else branch of this if statement?

```
        fn emit(self) {
            let reached_eof = if self.reached_eof { "true" } else { "false" };
            info!(
                message = "Stopped watching file.",
                file = %self.file.display(),
                reached_eof,
            );
            if self.include_file_metric_tag {
                counter!(
                    "files_unwatched_total", 1,
                    "file" => self.file.to_string_lossy().into_owned(),
                    "reached_eof" => reached_eof,
                );
            } else {
                counter!(
                    "files_unwatched_total", 1,
                    "reached_eof" => reached_eof,
                );
            }
        }
```